### PR TITLE
Various minor fixes for upgrade_admin_server.sh

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: scripts/upgrade_admin_server.sh [hunk 1]
```diff
@@ -36,6 +36,8 @@ upgrade_admin_server()
     bin/rake db:migrate
     bin/rake db:dump
     popd
+    chown crowbar:crowbar /opt/dell/crowbar_framework/db/schema.rb
+    chown crowbar:crowbar /opt/dell/crowbar_framework/db/data.yml
```
Comment (MaximilianMeister) [inline scripts/upgrade_admin_server.sh]: why not simply call:
```
sudo -u crowbar bin/rake db:migrate
sudo -u crowbar bin/rake db:dump
```

Comment (jsuchome) [inline scripts/upgrade_admin_server.sh]: That's also the option, but the result is the same

Comment (rsalevsky) [inline scripts/upgrade_admin_server.sh]: I prefer Maximilian's option.

**Comment by MaximilianMeister** [comment]
thanks for the update, just needs to be backported (a github label like `needs backport` would be nice)

+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**